src/main/drivers/dshot_bitbang_decode.c

```diff
@@ -142,7 +140,8 @@ uint32_t decode_bb_bitband( uint16_t buffer[], uint32_t count, uint32_t bit)
         return DSHOT_TELEMETRY_NOEDGE;
     }
 
-    int remaining = MIN(count - (p - b), (unsigned int)MAX_VALID_BBSAMPLES);
+    const uint32_t maxValidSamples = 69;
+    const uint32_t remaining = MIN(count - startMargin, maxValidSamples); // 60 is MAX_VALID_BBSAMPLES
```

Comment:
Probably no gain here, compiler is smart enough

But using `startMargin` may increase readability

Reply:
I'll keep it for readability.